docs/_documentations/troubleshooting.md

```diff
@@ -292,6 +292,17 @@ If you have the Codewind Language Server for Node.js Profiling extension enabled
 
 **Workaround:** Review the preceding list and ensure that your project conforms to all of the items in the list.
 
+<!--
+Action/Topic: Understanding application metrics
+Issue type: info
+Issue link: https://github.com/eclipse/codewind/issues/1251
+Info added in 0.10.0.
+-->
+## Setting Codewind server logging levels
+To assist with problem determination, change the default log level, such as trace, in PFE with the `cwctl loglevels` command.
```

Comment:
To assist with problem determination, raise the default Codewind server log level to Debug or Trace with the `cwctl loglevels` command.